client/me/reauth-required/security-key-form.scss

```diff
@@ -0,0 +1,21 @@
+.security-key-form__verification-code-form .button {
+	width: 100%;
+}
+.security-key-form__verification-code-form p {
+	margin-bottom: 0.5em !important;
```

Comment:
Any good reason to have the `!important` here? Perhaps if we use a classname it won't be necessary?

Reply:
I added it because the styles weren't being applied. Does it means we should use a class name for each `p` tag?

Reply:
Removed this in 003d81f92d

Reply:
Fixed this in 003d81f92d.

Reply:
Thanks - yeah, there is usually a better way to make it work than with `!important` - usually by adding a more specific selector.